### ISSUE TITLE
QueryResult.fields is empty, missing column name and type information

### ISSUE THREAD

**Issue by gthb** [oguimbal/pg-mem#234]
Having run `db.public.query('SELECT ...')` I get a `QueryResult`, and I want to read the column names and types for that result. But `QueryResult.fields` is currently always empty.

**Comment by gthb** [comment]
My current workaround for this:
```js
  /** @type {DataType[]} */
  // @ts-ignore (hacky use of internal `lastSelect` to workaround https://github.com/oguimbal/pg-mem/issues/234)
  const outputColumnTypeNames = DB.public.lastSelect.columns.map(c => c.type.primary);
```

**Comment by oguimbal** [comment]
Hi ! Fix released in `pg-mem@2.6.0`  :)
